llvm/unittests/Analysis/IR2VecTest.cpp

```diff
@@ -407,32 +360,148 @@ TEST_F(IR2VecTestFixture, GetFunctionVector) {
 
   EXPECT_EQ(FuncVec.size(), 2u);
 
-  // Function vector should match BB vector (only one BB): {1.29, 2.31}
-  EXPECT_THAT(FuncVec,
-              ElementsAre(DoubleNear(1.29, 1e-6), DoubleNear(2.31, 1e-6)));
+  // Function vector should match BB vector (only one BB): {44.4, 44.4}
+  EXPECT_TRUE(FuncVec.approximatelyEquals(Embedding(2, 44.4)));
 }
 
-TEST(IR2VecTest, IR2VecVocabAnalysisWithPrepopulatedVocab) {
-  Vocab InitialVocab = {{"key1", {1.1, 2.2}}, {"key2", {3.3, 4.4}}};
-  Vocab ExpectedVocab = InitialVocab;
-  unsigned ExpectedDim = InitialVocab.begin()->second.size();
+static constexpr unsigned MAX_OPCODES = 67;
```

Comment:
This constants currently don't follow the LLVM naming guidelines.

Reply:
I'm not sure if the LLVM Programmer's Manual/Coding Style/whatever other documents make a clear recommendation, but I definitely prefer `constexpr`. It plays a lot better with IDE tooling like clangd and is just easier to reason about in my opinion.